projects/cardiovascular-patient-app/src/app/report-bp/report-bp.component.spec.ts

```diff
@@ -38,16 +33,15 @@ describe('ReportBPComponent', () => {
   let component: ReportBPComponent;
   let fixture: ComponentFixture<ReportBPComponent>;
   let loader: HarnessLoader;
-  let fhirDataService: jasmine.SpyObj<FhirDataService>;
+  let matSnackBarSpy: jasmine.SpyObj<MatSnackBar>;
+
 
   beforeEach(async () => {
-    fhirDataService = jasmine.createSpyObj('FhirDataService', ['createBloodPressureResource', 'addPatientData']);
-    fhirDataService.createBloodPressureResource.and.returnValue(fhirResource);
-    fhirDataService.addPatientData.and.returnValue(Promise.resolve({}));
+    matSnackBarSpy = jasmine.createSpyObj('MatSnackBar', ['open']);
     await TestBed.configureTestingModule({
       imports: [ReactiveFormsModule, NoopAnimationsModule, MatInputModule, MatFormFieldModule, MatCardModule],
       declarations: [ReportBPComponent],
-      providers: [FormBuilder, { provide: DataLayerManagerService, useValue: mockLayerManager }, { provide: FhirDataService, useValue: fhirDataService }],
+      providers: [FormBuilder, { provide: MatSnackBar, useValue: matSnackBarSpy }],
```

Comment:
these tests are still using the real FhirDataService

Reply:
Okay, @semicolin  I will use FhirDataServiceSpy instead of the real FhirDataService.